lib/miq_pglogical.rb

```diff
@@ -65,6 +65,28 @@ def replication_lag
     self.class.with_connection_error_handling { pglogical.lag_bytes }
   end
 
+  def replication_type
+    if provider?
+      :remote
+    elsif subscriber?
+      :global
+    else
+      :none
+    end
+  end
+
+  def replication_type=(desired_type)
+    return unless PglogicalSubscription.logical_replication_supported?
```

Comment:
Side effects below... need the guard clause... I didn't want to risk `delete_all` running which also queues up work or could have new functionality in the future.